src/main/java/AddTask.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class AddTask {
+    public static boolean hasIndex(String[] words) {
+        try {
+            Integer.parseInt(words[1]);
+            return true;
+        } catch(Exception a) {
+            return false;
+        }
+    }
+
+    public static void addTask(Task[] tasks) {
+        int index = 0;
+        boolean isRunning = true;
+        while (isRunning) {
+            Scanner userInput = new Scanner(System.in);
+            String text = userInput.nextLine();
+            String[] words = text.split(" ");
+            switch (words[0]) {
+            case "bye":
+                isRunning = false;
+                break;
+            case "list":
+                PrintText.print(PrintText.HORIZON);
+                PrintTask.list(Arrays.copyOf(tasks, index));
+                PrintText.print(PrintText.HORIZON + "\n");
+                break;
+            case "mark":
```

Comment:
mark and unmark may not be related to addTask class, maybe you can create a taskListManager class that in charge of adding and marking.

Reply:
Thanks for your advice!